code/modules/admin/topic.dm

```diff
@@ -896,6 +901,16 @@
 		else
 			dat += "<td width='20%'><a href='?src=[REF(src)];[HrefToken()];jobban3=alien;jobban4=[REF(M)]'>Alien</a></td>"
 
+		//Other Roles (black)
+		dat += "<table cellpadding='1' cellspacing='0' width='100%'>"
+		dat += "<tr bgcolor='000000'><th colspan='5'><a href='?src=[REF(src)];[HrefToken()];jobban3=otherroles;jobban4=[REF(M)]' style='color: white;'>Other Roles</a></th></tr><tr align='center'>"
+
+		//Mind Transfer Potion
+		if(jobban_isbanned(M, ROLE_MIND_TRANSFER))
+			dat += "<td width='20%'><a href='?src=[REF(src)];[HrefToken()];jobban3=Mind Transfer Potion;jobban4=[REF(M)]'><font color=red>Mind Transfer Potion</font></a></td>"

```

Comment:
I know that the other job bans don't do this (don't worry about them), but you should use `[ROLE_MIND_TRANSFER]` here instead of `Mind Transfer Potion` (same above w/ the sentience potion).

Reply:
@spacemaniac Alright, will do.

Reply:
@spacemaniac should I so the same with sentience potions too?